src/modules/mana/utils.ts

```diff
@@ -92,3 +94,31 @@ export async function waitForSync(
     return waitForSync(txHash, isSynced)
   }
 }
+
+export const mapStatus = (txStatus: TxStatus | null) => {
```

Comment:
please use the helpers `isPending`, `hasFailed` and `hasSucceeded` from `decentraland-dapps` to map these statuses here, there are many more statuses that are not currently mapped (like Queued or Replaced), like:

```ts
if (isPending(txStatus)) {
  return TransactionStatus.PENDING
} else if (hasFailed(txStatus)) {
  return TransactionStatus.REJECTED
} else if (hasSucceeded(txStatus)){
  return TransactionStatus.CONFIRMED
} else {
  throw new Error(`Invalid transaction status: ${txStatus}`)
}

```

Reply:
Done thanks